### ISSUE TITLE
getIPV4Address failed when using PAIK8S mode

### ISSUE THREAD

**Issue by xuehui1991** [microsoft/nni#2008]
When using the PAIK8S mode, I got some error:

Training service error:
getIPV4Address() fail because os.networkinterfaces().eth0 is undefined.

By the way, I already set the "nniManagerIp" in config correctly.

**Short summary about the issue/question**:

**Brief what process you are following**: 

**How to reproduce it**: 

**nni Environment**:
- nni version: 1.3
- nni mode(local|pai|remote): paik8s
- OS: linux
- python version: .36
- is conda or virtualenv used?:  conda
- is running in docker?: yes

**need to update document(yes/no)**:

**Anything else we need to know**:

**Comment by SparkSnail** [comment]
This error is caused by the python environment. There are two NNI environment in user's machine, and one env is for 'local python', one is for 'virtual environment python', the two environment set nnictl's path to the same place, and the 'local python' nnictl path override the 'virtual environment python' nnictl path. When user use nnictl command, the system will always use 'local python' nnictl.

**Comment by ICEORY** [comment]
Hi, I also get the same error: getIPV4Address() fail because os.networkinterfaces().eth0 is undefined; I also test nni 1.2, and it works fine.

### log

```
[2/8/2020, 6:20:18 AM] INFO [ 'NNIManager setClusterMetadata, key: nni_manager_ip, value: {"nniManagerIp":"10.10.1.139"}' ]
[2/8/2020, 6:20:18 AM] ERROR [ 'Uknown key: nni_manager_ip' ]
```

```
{ Error: getIPV4Address() failed because os.networkInterfaces().eth0 is undefined.
    at Object.getIPV4Address (/usr/local/nni/common/utils.js:183:15)
    at PAIK8STrainingService.submitTrialJobToPAI (/usr/local/nni/training_service/pai/paiK8S/paiK8STrainingService.js:170:103)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  name: '',
  cause:
   Error: getIPV4Address() failed because os.networkInterfaces().eth0 is undefined.
       at Object.getIPV4Address (/usr/local/nni/common/utils.js:183:15)
       at PAIK8STrainingService.submitTrialJobToPAI (/usr/local/nni/training_service/pai/paiK8S/paiK8STrainingService.js:170:103)
       at process._tickCallback (internal/process/next_tick.js:68:7) }
```

### environment: 
- nni 1.3
- python 3.6
- openpai 0.14
- ubuntu 16.04
- running in docker 
- without anaconda

**Comment by SparkSnail** [comment]
Hi @ICEROY, this issue is fixed by https://github.com/microsoft/nni/pull/1987, will release this fix on v1.4.
